src/webui/src/static/interface.ts

```diff
@@ -184,10 +184,17 @@ interface ExperimentParams {
     };
     clusterMetaData?: {
         key: string;
-        value: string;
+        value: string | ClusterItem;
     }[];
 }
 
+interface ClusterItem {
+    command?: string;
+    codeDir?: string;
+    image?: string;
+    gpuNum?: number;
```

Comment:
There might be more items trial_config, like 'virtualCluster', 'memoryMB', 'paiStorageConfigName', will this work if more items added in trial_config?

Reply:
test, no problem